c2corg_ui/templates/outing/filters.html

```diff
@@ -1,104 +1,145 @@
+             uib-datepicker-popup="dd MM yyyy" is-open="openDateEnd" ng-change="filtersCtrl.setDate('date')">
+          <input type="text" disabled class="form-control" value="{{filtersCtrl.dates[1] | amDateFormat:'Do MMMM YYYY'}}" required/>
+          <span class="input-group-btn">
+            <button type="button" class="btn btn-default" ng-click="openDateEnd = true"><span class="glyphicon glyphicon-calendar"></span></button>
+          </span>
+        </div>
+      </div>
+
+      <div class="col-xs-12 col-sm-6 filter">
+        <label translate>elevation_max</label><br>
+        <app-slider filter="oalt" filters-list="filtersCtrl.filters" max="8850"></app-slider>
       </div>
 
-      <div class="col-xs-12 col-sm-6 col-md-4 filter">
-        <label translate>lang</label>
-        <select ng-model="searchFiltersCtrl.lang" class="form-control" ng-options="lang as (mainCtrl.translate(lang) |  capitalize ) for lang in ['${"','".join(default_langs) |n}'] | translate" ng-model="lang"></select>
+      <div class="col-xs-12 col-sm-6 filter">
+        <label translate>height_diff_up</label><br>
+        <app-slider filter="odif" filters-list="filtersCtrl.filters" max="5000"></app-slider>
       </div>
-      
-      <div class="col-xs-12 col-sm-6 col-md-4 filter">
-        <label>{{'summit'| translate | capitalize}}</label>
-        <input type="text" class="form-control">
+
+      <div class="col-xs-12 col-sm-6 filter">
+        <label translate>elevation_access</label><br>
+        <app-slider filter="oparka" filters-list="filtersCtrl.filters" max="5000"></app-slider>
       </div>
 
-      <div class="col-xs-12 col-sm-5 col-md-5 filter">
-        <label translate>height_diff</label><br>
-        <input type="range" class="range-between" id="height-filter" data-slider-tooltip="hide">
-        <span class="slider-min-max">
-          <p class="min-value">{{searchFiltersCtrl.height_diff_up.min}} m</p>
-          <p class="max-value">{{searchFiltersCtrl.height_diff_up.max}} m</p>
-        </span>
+      <div class="col-xs-12 col-sm-6 filter">
+        <label translate>frequentation</label><br>
+        <app-slider filter="ofreq" filters-list="filtersCtrl.filters" unit=""
+          values-list="['${"','".join(frequentation_types) |n}']"></app-slider>
       </div>
-      <div class="col-xs-12 col-sm-5 col-md-5 filter">
-        <label translate>elevation</label><br>
-        <input type="range" id="elevation-filter" class="range-between" data-slider-tooltip="hide">
-        <span class="slider-min-max">
-          <p class="min-value">{{searchFiltersCtrl.elevation.min}} m</p>
-          <p class="max-value">{{searchFiltersCtrl.elevation.max}} m</p>
-        </span>
+
+      <div class="col-xs-12 col-sm-6 filter">
+        <label translate>public_transport</label><br>
+        <a ng-click="filtersCtrl.toggleCheckbox('owpt')">
+          <input type="checkbox" ng-checked="filtersCtrl.filters.owpt">
+          <span></span>
```

Comment:
Yes, it's like in the screenshot.
I would prefer keeping the checkbox as is. I have already tried to put the checkbox and label on the same line but it looks weird since this criteria is misaligned with the others. I am not sure adding a new text is needed (and it will add one more string to translate among millions). Perhaps the current label should be updated to something more explicit.